listenbrainz/webserver/templates/user/import.html

```diff
@@ -29,6 +29,15 @@ <h3>Direct import from Last.fm</h3>
         </form>
       </div>
 
+    <h3> Import from Spotify </h3>
+
+    <p>
+      ListenBrainz can import future listens from your Spotify account.
```

Comment:
I think this is a bit confusing, I guess you're trying to say that it can't import the past listening history (or did I write this?)
How about:
> ListenBrainz can automatically import songs from Spotify as you listen to them